src/Apps/Settings/Routes/MyCollection/Components/EmptyMyCollectionPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { Spacer } from "@artsy/palette"
+import { HowMyCollectionWorks } from "./HowMyCollectionWorks"
+import { MyCollectionAppDownload } from "./MyCollectionAppDownload"
+import { MyCollectionBenefits } from "./MyCollectionBenefits"
+
+export const EmptyMyCollectionPage: React.FC = () => {
+  return (
+    <>
+      <MyCollectionAppDownload />
+      <Spacer my={12} />
+      <MyCollectionBenefits />
+      <Spacer my={12} />
+      <HowMyCollectionWorks />
+    </>
```

Comment:
A slightly nicer way to do this would be to use the [Join component](https://palette.artsy.net/elements/join/)